products/billingbudget/api.yaml

```diff
@@ -0,0 +1,187 @@
+# Copyright 2019 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: Billing
+display_name: Billing Budget
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://billingbudgets.googleapis.com/v1beta1/
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Billing Budget
+    url: https://console.cloud.google.com/apis/library/billingbudgets.googleapis.com/
+objects:
+  - !ruby/object:Api::Resource
+    name: Budget
+    base_url: billingAccounts/{{billing_account}}/budgets
+    self_link: '{{name}}'
+    update_verb: :PATCH
+    # TODO: re-enable this. When this is enabled, changes are not being made
```

Comment:
Does anything need to be done here? When will we know to apply this TODO?

Reply:
I haven't looked too deeply into it, but in my testing when this was enabled no changes were properly being applied... I am guessing the update mask + flattening don't mix well together. I would enable this again once the API is fixed and dig deeper then.